incubator/binding-kafka-grpc/src/main/java/io/aklivity/zilla/runtime/binding/kafka/grpc/internal/stream/KafkaGrpcRemoteServerFactory.java

```diff
@@ -123,6 +125,7 @@ public final class KafkaGrpcRemoteServerFactory implements KafkaGrpcStreamFactor
     private final BufferPool bufferPool;
     private final MutableDirectBuffer writeBuffer;
     private final MutableDirectBuffer extBuffer;
+    private final MutableDirectBuffer metadataBuffer;
```

Comment:
We can remove this `metadataBuffer` per our discussion about upcoming `INIT_DEFERRED` flag for `DATA` frames.

Reply:
The one we discussed was about GrpcKafka this is for `KafkaGrpc`

Reply:
The one we discussed was in GrpcKafka, this is for KafkaGrpc to filter metadata that's not zilla specific.